dbms/src/Debug/astToExecutor.cpp

```diff
@@ -1561,11 +1666,100 @@ ExecutorPtr compileExchangeSender(ExecutorPtr input, size_t & executor_index, ti
     return exchange_sender;
 }
 
-
 ExecutorPtr compileExchangeReceiver(size_t & executor_index, DAGSchema schema)
 {
     ExecutorPtr exchange_receiver = std::make_shared<mock::ExchangeReceiver>(executor_index, schema);
     return exchange_receiver;
 }
 
+ExecutorPtr compileWindow(ExecutorPtr input, size_t & executor_index, ASTPtr func_desc_list, ASTPtr partition_by_expr_list, ASTPtr order_by_expr_list, mock::MockWindowFrame frame)
+{
+    std::vector<ASTPtr> window_exprs;
+    std::vector<ASTPtr> order_columns;
+    std::vector<ASTPtr> partition_columns;
+    DAGSchema output_schema;
```

Comment:
Seems like the schema is wrong?
schema = input->schema + window functions

Reply:
Wow, I forget to change it! Thanks!